configure.ac

```diff
@@ -49,7 +49,7 @@ PKG_CHECK_MODULES(INITIAL_SETUP,
                   pango >= $PANGO_REQUIRED_VERSION
                   rest-0.7
                   json-glib-1.0
-		  libsecret-1
+                  libsecret-1
```

Comment:
@mcatanzaro I saw you pushed the patch to upstream master. How should I update the PR?
Should I push those commit on our gnome-3-22 branch and rebase on top of that?